docs/tasks/benchmarking.md

```diff
@@ -29,7 +29,7 @@ nuctl deploy helloworld-go -n nuclio -p https://raw.githubusercontent.com/nuclio
 
 Run the benchmark:
 ```sh
-wrk -c 36 -t 36 -d 10 http://172.17.0.1:39150
+>>> wrk -c 36 -t 36 -d 10 http://172.17.0.1:39150
```

Comment:
no need in `>>>` . You can check how it looks locally with `make -C docs/ make html` :
![image](https://github.com/nuclio/nuclio/assets/35141662/e7ba2202-d920-4bb9-847e-e9f4ebc07e5d)